index.html

```diff
@@ -2,38 +2,64 @@
 <html>
 <head>
     <meta charset="utf-8">
-    <title>What should I do?</title>
-    <link rel="stylesheet" href="style.css">
+    <title>Abbas Mashhouri Component</title>
+    <link rel="stylesheet" href="css/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
 </head>
-    <body>
-        <script type="text/x-handlebars" data-template-name="todos">
-            <section id="todoapp">
-                <header id="header">
-                    <h1>The TODO list</h1>
-                </header>
+<body>
+    <h1 class="mainTitle"> Task Component</h1>
+    <script type="text/x-handlebars" data-template-name="Abs">
+        <section id="TasksSection">
+            <header id="header">
```

Comment:
why did you name some "id" whit capital letters and other not? E.g TasksSection and header